addons/payment/security/ir.model.access.csv

```diff
@@ -5,9 +5,7 @@ payment_provider_system,payment.provider.system,model_payment_provider,base.grou
 payment_icon_all,payment.icon.all,model_payment_icon,,1,0,0,0
 payment_icon_system,payment.icon.system,model_payment_icon,base.group_system,1,1,1,1
 payment_token_all,payment.token.all,model_payment_token,,1,0,0,0
-payment_token_portal,payment.token.portal,model_payment_token,base.group_portal,1,1,1,1
-payment_token_system,payment.token.system,model_payment_token,base.group_system,1,1,1,1
-payment_token_user,payment.token.user,model_payment_token,base.group_user,1,1,1,1
+payment_token_system,payment.token.system,model_payment_token,base.group_system,1,0,1,1
```

Comment:
We'll need to keep the write permission for administrators because, right now, they can't archive tokens from the backend. However, the view could be made readonly since that's what we wanted to do in the first place.